src/components/CurrentScenarioPowerBIReport/CurrentScenarioPowerBiReportHook.js

```diff
@@ -14,21 +14,20 @@ import { getReportLabels } from './labels';
 export const useCurrentScenarioPowerBiReport = () => {
   const { t, i18n } = useTranslation();
 
-  const currentScenarioData = useCurrentScenarioData();
-  const scenarios = useScenarios();
+  const currentScenarioData = useCurrentSimulationRunnerData();
+  const scenarios = useRunners();
   const reports = usePowerBIInfo();
   const logInWithUserCredentials = useWorkspaceChartsLogInWithUserCredentials();
-  const downloadLogsFile = useDownloadLogsFile();
+  const downloadLogsFile = useDownloadSimulationLogsFile();
 
   const reportLabels = useMemo(() => getReportLabels(t), [t]);
   const language = useMemo(() => i18n.language, [i18n.language]);
 
   const visibleScenarios = useMemo(
     () =>
-      scenarios.map((scenario) => ({
-        id: scenario.id,
-        runId: scenario.lastRun?.scenarioRunId,
-        csmSimulationRun: scenario.lastRun?.csmSimulationRun,
```

Comment:
As I understood, the run property replaces `csmSimulationRun` from back-end point of view, can it be done in powerBi filter as well or it's trickier than that ?

Reply:
Indeed, it seems that the [Run](https://github.com/Cosmo-Tech/cosmotech-api/blob/main/doc/Models/Run.md) object (replacing the ScenarioRun) still has a property `csmSimulationRun`, so we should try to keep supporting the `csmSimulationRun` option.

In the last line of the highlighted code, we're removing the `csmSimulationRun` property, is there a reason preventing us to keep it?

Reply:
I deleted it because runner object has only `lastRunId` property which is equal to `csmSimulationRun` and I didn't think about powerBi :woman_facepalming: if we keep both properties in runner object in Redux, will it help to keep filters or we'd better update filters to use `lastRunId` key ? :thinking:

Reply:
Good question, we can ask the PO :smile: 
Maybe keeping the two filters could help to decrease the cost of version upgrade. Yet we should probably define one of the two filters as deprecated, so we can remove it in the future